DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EmailDelegatesController.cs

```diff
@@ -14,6 +15,7 @@ namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates
     using Microsoft.AspNetCore.Authorization;
     using Microsoft.AspNetCore.Mvc;
     using Microsoft.FeatureManagement.Mvc;
+    using ConfigHelper = DigitalLearningSolutions.Web.Helpers.ConfigHelper;

```

Comment:
Aren't we trying to transition the ConfigHelper to the Data project? So any new references should be to DLS.Data, not Web.